fxa-oauth-server/test/lib/mocks.js

```diff
@@ -50,7 +50,18 @@ function requireDependency(dependency, modulePath, basePath) {
     );
   }
 
-  return require(localPath);
+  try {
+    return require(localPath);
```

Comment:
this passes for https://github.com/mozilla/fxa-auth-server/blob/master/fxa-oauth-server/test/db/mysql.js#L21 because the mysql module is `mysql/index.js`, but for `./token.js` the resolve needs to be adjusted.